src/main/java/net/openhft/chronicle/wire/JSONWire.java

```diff
@@ -400,12 +400,8 @@ private Object parseType() {
                 final StringBuilder sb = Wires.acquireStringBuilder();
                 sb.setLength(0);
                 this.wireIn().read(sb);
-                try {
-                    final Class<?> clazz = classLookup().forName(sb.subSequence(1, sb.length()));
-                    return parseType(null, clazz);
-                } catch (ReflectiveOperationException e) {
-                    throw new RuntimeException(e);
-                }
+                final Class<?> clazz = classLookup().forName(sb.subSequence(1, sb.length()));
+                return parseType(null, clazz);
```

Comment:
No need to rethrow as RuntimeException now, it will just throw a more specific RuntimeException